accessibility/alt-text.md

```diff
@@ -167,4 +170,28 @@ However, you MUST be careful with this - not everyone is comfortable with having
 * [Writing great alt text: Emotion matters](https://jakearchibald.com/2021/great-alt-text/)
 * [Thoughts on skin tone and text descriptions](https://tink.uk/thoughts-on-skin-tone-and-text-descriptions.md-notes-on-synthetic-speech/), by Léonie Watson
 
+### Images of text 
 
+<img width="625" alt="screenshot of social media post of an image of text" src="https://github.com/springernature/frontend-playbook/assets/69847167/83772c8e-6b61-4f5a-8970-fc91be79f84d">
+
+In a Springer Nature social media post, this image of text and book covers was accompanied with the copy: “Are you a horror enthusiast interested in finding out more about vampires in film? Then this book is a must-read for you! From the first ever filmic female vampire to a Dracula look-alike and cinema’s original vampire, this book discusses them all. https://bit.ly/3SnsTrN” 
+
+Images of text are often used on social media and in advertisements for visual appeal and to convey information quickly. But images are one of the least accessible ways of presenting text - if a user needs to magnify or zoom their screen, the images can become distorted or pixellated, rendering the text unreadable. If the image is complex, like this one, then it’s much harder to write sufficient alt text for it. You should avoid publishing images of text as much as you can. If it’s unavoidable, use as little text as possible, and don’t make the text in the image your only call to action.
+
+This example is a complex graphic with a lot happening in it, and it’s easier to start with what you shouldn’t do with an image like this:
+
+#### Bad alt text
+
+* “text and book covers”
+* “Vampires on the silent screen” 
+* “Image of book cover”
```

Comment:
Perhaps some explanation as to why they are "bad" would be helpful?

Reply:
I think the guiding principles section covers why this would be insufficient information

A few guiding principles

- Don’t say ‘Image of…’ or ‘Picture of…’ etc. (It is nearly always wrong to say an "Image of", because all images are … images. It is extraneous information that the screenreader announces. However, it can help to know the type of image - i.e. is it a photo, or a screenshot, or a cartoon, etc, depending on the surrounding content and context.)
- Don’t need to repeat information already given in the text on the page.
- Be as succinct as possible but include details as needed. There is no hard character limit despite some guidance stating otherwise, a screen reader won't truncate alt text.
- Copyright info, image source, or other extra information should go in the caption, not the alt text.
- Context is important e.g. it may be appropriate to note if an image is a photo or an artwork, whether a person is old or young, if the sky is stormy or clear, if that is relevant to the content.
- Think about how you would describe the image to a person who can’t see it.


Do you think so or is it worth adding more?

Reply:
The other examples have an explanation underneath describing _why_ these alt texts are insufficient, whereas these do not.